lib/shopify_transporter/exporters/magento/soap.rb

```diff
@@ -74,12 +74,17 @@ def call_with_retries(method, params, retry_count = 0)
 
         def soap_client
           @soap_client ||= Savon.client(
-            wsdl: "#{@hostname}/api/v2_soap?wsdl",
+            wsdl: "#{hostname_with_scheme}/api/v2_soap?wsdl",
             open_timeout: 500,
             read_timeout: 500,
           )
         end
 
+        def hostname_with_scheme
+          uri = URI.parse(@hostname)
+          uri.scheme.present? ? @hostname : 'https://' + @hostname
```

Comment:
I wish I could just do something like this instead:

```
def hostname_with_scheme
          uri = URI.parse(@hostname)
          uri.scheme ||= 'https'
          uri.to_s
end
```

But unfortunately that doesn't work. The URI library doesn't do the `//` thing, so you end up getting `https:example.magento.com`. I'm open to suggestions on how to get this method cleaner.